docs/hub/repositories.md

```diff
@@ -17,7 +17,7 @@ In these pages, you will go over the basics of getting started with Git and Xet
 - [Getting Started with Repositories](./repositories-getting-started)
 - [Settings](./repositories-settings)
 - [Storage Limits](./storage-limits)
-- [Storage Backends](./storage-backends)
+- [Storage Backend - Xet](./storage-backends)
```

Comment:
```suggestion
- [Storage Backend (Xet)](./xet)
```

maybe we take this opportunity to just rename that entry point page to /docs/hub/xet